tests/test_binary.nim

```diff
@@ -104,5 +115,13 @@ proc main =
           outp.contains("'-o, --offline' was given without passing '-p, --prob-specs-dir'")
           exitCode == 1
 
+  suite "uuid":
```

Comment:
When does a test go in the `test_binary.nim` file and when in a separate file like `test_probspecs`? It feels like maybe you would want both here, with `test_uuid` containing uuid specific tests?

Reply:
Yes, that's a question we'll keep facing. It's partly a philosophical question of integration tests vs unit tests.

We can test our code with unit tests, but it doesn't guarantee that the binary actually uses the code that's being tested.

I've added a test of `genUUID` to `test_uuid.nim` though - any thoughts?

Reply:
I like it